catalog-info.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: pandablocks
+  title: PandABlocks-client
+  description: A Python client to control and data ports of the PandABlocks TCP server
+spec:
+  type: documentation
+  lifecycle: experimental
+  owner: user:default/tmc43
```

Comment:
Please can you change the answer to the owner to be `group:default/sscc`. Actually, can you make this the default in the copier template too please.